src/common-signals.js

```diff
@@ -27,24 +27,35 @@ export const CommonSignals = {
   BUILT: 'built',
 
   /**
-   * The initial contents of an element/document/embed have been loaded.
+   * The element has started loading.
+   * LOAD_START triggers at the start of the layoutCallback.
    */
-  INI_LOAD: 'ini-load',
+  LOAD_START: 'load-start',
 
   /**
-   * The element has been loaded.
+   * Rendering has been confirmed to have been started.
+   * RENDER_START is an optional signal, implemented by ads, shadowdoc.
```

Comment:
be more future looking. explain what extension should consider implement this signal, and what feature should listen to it. ads & shadowdoc can be 2 current examples

Reply:
The way i look at most of these signals - they apply to mostly everything. Just not always interesting for simpler elements. Basically, a generic component's render-start is as soon as `buildCallback` has been called. Since there are many ways to look for build info, it hasn't been necessary to extend render-start beyond more complicated elements.

Reply:
> The way i look at most of these signals - they apply to mostly everything. Just not always interesting for simpler elements

I agree. But what exactly does RENDER_START mean? Is it reasonable for me to assume that it's related to "something will show up". If that's the case I find it to be more related to `layoutCallback`.

Reply:
It's more of a meaningful display sort of thing. E.g. text could be displayed already. CSS is correctly installed/applied. But resources are not yet loaded. It's more similar to this: https://twitter.com/cramforce/status/1118743321558446080?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed&ref_url=https%3A%2F%2Fd-363929593147740339.ampproject.net%2F1905091827220%2Fframe.html

Reply:
That makes a lot of sense!